app/presenters/completions_presenter.rb

```diff
@@ -120,6 +120,10 @@ def pii
     end
   end
 
+  def multiple_emails?
+    current_user.email_addresses.count > 1
```

Comment:
As mentioned in an earlier review, we should only ever consider confirmed email addresses.

```suggestion
    current_user.confirmed_email_addresses.any?
```